### ISSUE TITLE
pulumi complains about changes to unmodified resources 

### ISSUE THREAD

**Issue by ocobles** [equinix/pulumi-equinix#90]
Pulumi tries to recreate some resources without modifications, like the RoutingProtocol resource due to changes in the "name" field even though there are no changes. Although it could be temporarily suppressed with ignore_changes=["name"] we must investigate and identify if we are missing something in the configuration or if there is a problem in the pulumi terraform bridge plugin

```
routing_protocol_bgp_gcp = equinix.fabric.RoutingProtocol("rp-bgp",
    opts=pulumi.ResourceOptions(
        depends_on=[routing_protocol_direct_gcp],
        ignore_changes=["name"]
    ),
    type="BGP",
    connection_uuid=fabric_connection_gcp.id,
    name="FabricToGCPRoutingProtocolBGP",
    customer_asn=gcp_peer_config.gcp_asn,
    bgp_ipv4=equinix.fabric.RoutingProtocolBgpIpv4Args(
        customer_peer_ip=gcp_peer_config.gcp_router_ip
    ),
)
```

**Comment by ocobles** [comment]
While the issue with field `name` mentioned above will be fixed in https://github.com/equinix/pulumi-equinix/pull/91 (it was an issue in terraform provider), there are some other resources and fields with that issue that I think it could be a problem with the bridge plugin https://github.com/pulumi/pulumi-terraform-bridge/issues/1951

**Comment by ocobles** [comment]
fixed in #104
